content/chat/rooms/message-reactions.textile

```diff
@@ -0,0 +1,192 @@
+---
+title: Message reactions (experimental)
+meta_description: "React to chat messages"
+product: chat
+languages:
+  - javascript
+  - react
+---
+
+<aside>
+<p>The Message Reactions feature is new and <strong>experimental</strong>.</p>
+</aside>
+
+Add, remove and display message reactions in a chat room. Users can react to messages, typically with emojis but can be any string, and others can see the reactions to the message. Message reactions can be added and removed and a summary of the reactions is persisted with the message.
+
+The reactions can be any strings. Summaries are aggregated based on unique strings. UTF-8 emojis are a common use case, but any string can be used as long as they are consistent across all front-ends of your app. Examples of common reactions are @👍@, @❤️@, @:like:@, @like@, @+1@, and so on. How those are presented to the user is entirely up to the app.
+
+h2(#adding-reactions). Adding a message reaction
+
+To add a message reaction use @room.messages.reactions.add(message, params)@. This method takes the following parameters:
+- @message@ - The message to add the reaction to. It can also be an object of format @{serial: "message serial"}@.
+- @params@ - Set the @reaction@ (for example the emoji), and optionally override the @type@ or set a @count@.
+
+```[javascript]
+    // Add a 👍 reaction using the default type
+    await room.messages.reactions.add(message, { reaction: '👍' });
+
+    // The reaction can be anything, not just UTF-8 emojis:
+    await room.messages.reactions.add(message, { reaction: ':like:' });
+    await room.messages.reactions.add(message, { reaction: '+1' });
+    
+    import { MessageReactionType } from '@ably/chat';
```

Comment:
It feels that this is in the middle of a codeblock, I don't think you'd ever see this in practice - can we move to the top?